react-redux/react-redux-tests.tsx

```diff
@@ -34,14 +40,23 @@ class Counter extends Component<any, any> {
     }
 }
 
+class Counter extends Component<CounterProps, any> {
+    render() {
+        return (
+            <button onClick={this.props.onIncrement}>
+                {this.props.value}
+            </button>
+        );
+    }
+}
 function mapStateToProps(state: CounterState) {
     return {
         value: state.counter
     };
 }
 
 // Which action creators does it want to receive by props?
-function mapDispatchToProps(dispatch: Dispatch<CounterState>) {
+function mapDispatchToProps(dispatch: Dispatch<Action>) {
```

Comment:
This change is not correct -- `dispatch` is parameterized by the state it's relevant to, not by any action.